src/include/iscsi-lio-server/widgets.rb

```diff
@@ -643,6 +643,9 @@ def AddClntDialog
           if @changed_lun.has_key?(s)
             txt = _("Client name already exists!")
           end
+          if s.index /["\*\?"]/
```

Comment:
Question why do you want to find position when you do not use it? If you want to add to text position of wildcard, then it will be fine, but when you want just test if string contain such characters, then index of findfirstof is useless. I think there was no good builtin in old ycp for such task.